examples/opensora_pku/opensora/dataset/transform.py

```diff
@@ -1,4 +1,5 @@
 import html
+import random
```

Comment:
add random.seed at set_seed function

Reply:
There is `random.seed` set in `set_random_seed`, imported from `mindone.utils.seed`